### PR TITLE
Wb gen sw

### PR DESCRIPTION
**Pull request by gauntface**
Fixes: https://github.com/GoogleChrome/workbox/issues/1208

Adds guides on generating a complete service worker with Workbox.

### CODE REVIEW

File: src/content/en/tools/workbox/guides/_shared/register-sw.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{# wf_md_include #}
+
+<pre class="prettyprint html">
+&lt;script&gt;
```
Comment (jeffposnick) [inline src/content/en/tools/workbox/guides/_shared/register-sw.md]: It looks like there's only an opening `<script>` without a closing tag. Unless that gets inserted by another partial template?

File: src/content/en/tools/workbox/guides/generate-service-worker/workbox-build.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  return workboxBuild.generateSW({
+    globDirectory: 'build',
+    globPatthers: [
+      '**\/*.{html,json,js,css}',
+    ],
+    swDest: 'build/sw.js',
+  });
+}
+
+buildSW();
```
Comment (jeffposnick) [inline src/content/en/tools/workbox/guides/generate-service-worker/workbox-build.md]: Could you remove this `buildSW()` invocation? It kind of runs counter to your earlier comment about needing to invoke the function *after* your assets are built, and you're also not making use of the promise returned by `buildSW()`.

**Comment by gauntface** [comment]
cc @jeffposnick

**Comment by petele** [comment]
@jeffposnick - please "squash and merge" when you're happy.

**Pull request merged**